examples/RNOneSignal/App.js

```diff
@@ -66,80 +66,81 @@ export default class App extends Component {
             isEmailLoading: false,
 
             // In-App Messaging states
-            iam_paused: false,
-
-            // Add more states here...
+            iamPaused: false,
 
             // Demo App states
             debugText: ''
+
+            // Add more states here...
         };
 
-        // Log level logcat is 6 (VERBOSE) and log level alert is 0 (NONE)
-        OneSignal.setLogLevel(6, 0);
+    }
 
-        // Share location of device
-        OneSignal.setLocationShared(true);
+    /* R E A C T  L I F E C Y C L E */
 
+    async componentDidMount() {
+        console.log("Mounted!");
+        /* O N E S I G N A L  S E T U P */
+        // Log level logcat is 6 (VERBOSE) and log level alert is 0 (NONE)
+        OneSignal.setLogLevel(6, 0);
+        OneSignal.setAppId(appId);
         OneSignal.setRequiresUserPrivacyConsent(requiresPrivacyConsent);
-        OneSignal.init(appId, {
-            kOSSettingsKeyAutoPrompt: true,
+        OneSignal.setLocationShared(true);
+        OneSignal.promptForPushNotificationPermissions(prompt => {
+            console.log("OneSignal permission:", prompt);
+        })
+
+        /* O N E S I G N A L  H A N D L E R S */
+        OneSignal.setNotificationWillShowInForegroundHandler(notifReceived => {
+            console.log("OneSignal: notification will show in foreground:", notifReceived);
+            setTimeout(()=>notifReceived.complete(notifReceived), 0);
```

Comment:
Should be `notifReceived.complete(notifReceived.getNotification())`

Reply:
Resolved  w/ c01f866